shuffle/move/examples/e2e/message.test.ts

```diff
@@ -27,33 +27,52 @@ Deno.test("Ability to set message", async () => {
 });
 
 Deno.test("Ability to set NFTs", async () => {
+  let initialize_txn = await main.initializeNFTScriptFunction(context.defaultUserContext);
+  initialize_txn = await devapi.waitForTransactionCompletion(initialize_txn.hash);
+  assert(initialize_txn.success);
+
   const contentUri = "https://placekitten.com/200/300";
   let txn = await main.createTestNFTScriptFunction(contentUri);
```

Comment:
As discussed in slack, let's exclusively use `invokeScriptFunction(...)` rather than codegen to simplify the tutorials at a holistic level.

Reply:
createTestNFTScriptFunction uses invokeScriptFunction, just a wrapper